src/main.css

```diff
@@ -154,6 +154,11 @@
CsIAgpqEARQlQU1IgKUbuBIk0o3KRZRbixkEyxzTbbrIRGiG1GSNOMkBYxnHvfm8/OjndnHTs7mtn3vffcz7vvvhmyiEjREt2g3I2b792hkS/uf/2ApqmAdvJ94v6obFHu88++ekAj+olmS+4C5fCfI5W/hNotmXGMvqcf6Gd6Qr/Rn/S3VbBmrSVrzfrU+sb6xfrd+sP6y/onN5a7klvLfZn7NreRq+ac3H/5Ql7lS/kF0DlGh4FtGHxGaYwmaIrKQHaUXqQZmkXvHM3TSVqgM3SWlukSXaYVWqXr9DbdoXW6Cwpl3wOWSd/FPEUzqB3DPUgTeE76DbQ00eLSil+ldd+BBNOozWDsEE1SEbW61GZ5DP5XMWadShhXwIzbQDEAOh5Nob+MvhnfBr0WOAyjdQIlpvciysx7DuUCjcp4HusKmiFIWUSrDUoOejxgLZleR8Y7wkOjd0F7FE+mPYk5U9Kjxw6Dqw1aCvoqaRlRnkabIEB5DuV5jDyF/gX8DwktLQHTcQTRHO5TuOOyRb2WICkY7o6Ruw5ck3hOobUsmq1CaxrxYTM20tEs0DOtFeh/FfW7wm1UWjy0eLCGh1aXDkGmhpFJAbcC7obYYh1z74Jjnq76NbqG2j3UhjDewdiyGesZu7mwG/tSAbT4GgO/ElrHYcPTGLWEMvN9E5p5ByNv0zX6GGWmVze8WWd10CsadE1QYzuzhVib6+i16ANQtGhNvISpM6WCQVUSRIzEkpm58hNeK+V/p3+lN+giVpnnu3J5fNHB/hT41XzHb/p1v+o/Br8iNKlQa6DVEQwN3/Y3gV3hv4q22l55Qb9KOIIT21NLBw4Ot5ryj4wBJVvain1zKcrsmuivCU7b4FqEVRTKNdQ0TwcSPoQkJR7D9QBNvz9GD+q63OS6LoGbF2utmrILTv3z2PG3DWon5GpnxdevP2BOHV5awn/F/8nYTbjjUTJU41JwW4ml7Eumuujdi+swC1bxlv416GYa5XRS3wu3LJw6pc2GsV9k4n31RJvdr49nGmXseeAxa7+4zHbt/fBARSiJv8fs4m/oKBhEimg17rud7M4o1PfqdZ6ZlzezYujg5PUrk1hAIS+JYUUkb9+fNkFX6R1jz/5rp2MD9Xobb2c/VkqWWJxuq8xcqnyJruqhLpup0aaInkrcD4RvFTtNva+cJXUXaG8zNZVcSf1o1WRGXrs3S6sTUlNBD1rddmtm2tEkz2q31u7zOnt4NfdjOYy2QzvZoUx2kLHo3EJbKco5TA7gZtun/S1cVcnDdoJ1HPJ0I71Iblb1N4LdN9yxduARO1Hu0UMaTW1A10SCFIQ4MzByFeOdiEfdo6DOyZPUdbTQegnpqsiikWWz71pB5i8ngdbedkKNNWO2/BjWqiA35nXI/lyR/aiCW5Cg9zHKm+iBVaHvrZBrETuZzNSZeq/MwujcC+JDqp1so1Ov0+MzyaREpqYgd00mrsBP8aoNvZAxbPNZ2fh75BlyMhGEvWQqSiyqALEtMXBL6FUwfxvlTaHCUWALZ40tOYFUgagW0x5LW8cMzanWRSYV9yeNN7S0ndg1VHKMcPLCbL2HHkUfrV6nT4l6tW7em2HXeESzqacac86gB+nZkNCvdkZ80WYabhWjmZ5bvd/rVNSZw3VqEvz1zqDtpbrkcNG+pAKfDS2teucWWI/bxvrNXeOX6pGtNxN2q6btwrLy9Xphb7allmG3DuIy4wzWnc5u0WNOtEK3EV9hutw96ul+jBzQUb9NhlbmvXCZY0L83I/VcxlIW2JBHRlb8sZgmQZia8nMNvVrQsnr5jU6r0zML4U9RG+1rfPFjpHR3Nl+T8fGBi7vfrEY0eidM3bfebHOWnxl8YNUnK2s8VziZbZoVN3lTO9miEYbJgupsH+L7RuhjzpGijNBxDFjbbOfcbx8iHszoaEKaNT5akO5qfdPuXhNbQslp9ceJ2uvZfRh7GzWE6wbys67XMNQ0W/b6trLgeY7eb9nJ/eaMANzwzdA0V6hOnxDBahT8mmVjNIxLzeZoaYr7wLjv3LazF000dhtnSeisWc04nXuUH28e3PkbSHnRl6gt9iOaweaROlRcmXpfDR6zxV6ZD3Id6NVJrVWuge3vUPV7w+8xLvVVmaJHG3nxDnXlbgXZA4tkTbtnFvS569MMeLpTtTtnJx9fc+5L2+hohy2w8MazwDVmax7/V4idBdOdICc9vUNay9Peer3iDlovUCDKB3ClZfvlAUapRM0RAt0HTHtBt2k1+gWvUuX6Dat0evIMz9CznCP7qOVv5p+QpdpjlZB41VDdQClI/gfpCU6D92fpwtYC8P0vHyD4i+FI+D2HJ3GFfyGaBzPCTpKk6A6RVdwXZVvpVw6CqT8zZS/mObpOLieopPh3EWaB1qil+hlegU1Asqz0r5I59C3glznIjCPQYYXMFP/TtAMJNMSDILyEGpT0nMclKbEa+aAZgKoJ4DjHGgvQ7LzkGoR9C+A24h8P+bfAugeinmx/j/c5t188xfhEyI7QZIjpncY2Mahl3HoqIDyGHDyk79ijOPGKVXkn4dutdxHxHozVP4fB0zPxgAAAQAAAAoACgAKAAB42oVSvU7DMBCem6c4eWoH4qRCAiEnBVWiILFRxHyyr4nV2K4ct7Q8PSYkFaQD3s73/Z3PYnE0DRzIt9rZguVpxoCsdErbqmBv68erW7YoE2EooMKAf6FlAvGIA1nlPFg0VLClM4a81NjA+rQjBnvfFKwOYXfHuTw3Q+ylsWTAe5VGS7It/YPnPYx1pPPR6sIZXnpkmUwmItAxlEtnA8oA2m7c/aU2bOIUQ4xvjDcY4rCp4B39J+eQoC+l2528ruq+3d116OlyNs/yOYxipfDQNNAxWvDUkj+QGjmMNEXwqMig3449Vsq+E23j+8MToQLdAkbRSreBPCk4E8FtxkH4q6w/0IdP6eDZylGEX5aTYTXUr3hdE6z26JVGC/l1dpPlMJ11mxR8+CnlF4RXweM=);
```

Comment:
Can these go into their own css files please?

Reply:
This is how Akash did it...also its set up as csss not SCSS

Reply:
I mean rather than have a big old base64 encoded string, we could put them in another file and link them from the html.